inst/@sym/display.m

```diff
@@ -75,14 +75,9 @@ function display(x)
   else
     unicode_dec = false;
   end
-  if (exist('OCTAVE_VERSION', 'builtin') && ...
-      compare_versions (OCTAVE_VERSION (), '4.3.0', '>='))
+  if (exist('OCTAVE_VERSION', 'builtin'))
     [fmt, spacing] = format();
     loose = strcmp (spacing, 'loose');
-  elseif (exist('OCTAVE_VERSION', 'builtin') && ...
-      compare_versions (OCTAVE_VERSION (), '4.0.0', '>='))
-    % Octave 4.1 dropped (temporarily?) the get(0,...) approach
-    loose = eval('! __compactformat__ ()');
   else
     % Matlab and Octave < 4
     loose = strcmp(get(0, 'FormatSpacing'), 'loose');
```

Comment:
As per the docstring of `[FORMAT, FORMATSPACING] = format` in Octave 5.1 https://hg.savannah.gnu.org/hgweb/octave/file/d05d6eebde10/libinterp/corefcn/pr-output.cc, `loose` is the default format for displaying matrices. We should be able to drop the `eval` stuff w/o any issues.